api/v1alpha1/capp_types.go

```diff
@@ -37,6 +37,13 @@ type CappSpec struct {
 	// +optional
 	Site string `json:"site,omitempty"`
 
+	// State defines the state of capp
+	// Possible values examples: "enabled", "disabled".
```

Comment:
This is redundant since there's a validation marker to tell us the possible values